scylla-server/tests/dataTypes-services.test.ts

```diff
@@ -0,0 +1,28 @@
+import prisma from '../src/prisma/prisma-client';
+import { describe, test, expect } from 'vitest';
+import { getAllDataTypes } from '../src/services/dataTypes.services';
+
+/**
+ * Tests for CRUD service functions
+ */
+
+/**
+ * unit tests for get all dataTypes
+ * (may or may not be hijacking dylan's tests D:, thank you Dylan.)
+ */
+
+describe('Data Type', () => {
+  test('Get All Data Types Works', async () => {
+    const expected = [
+      { id: 1, dataType: 'string' },
+      { id: 2, dataType: 'int' }
+    ];
+    const prismaMock = require('../prisma/prisma-client');
+    prismaMock.dataType.findMany.mockResolvedValue(expected);
```

Comment:
no you dont need to mock these because were using an sqlite db which is included in github so we actually wanna test our prisma calls